### ISSUE TITLE
Non-user directory installation pop-up in the middle of installation is annoying

### ISSUE THREAD

**Issue by k-rus** [Starcounter/Starcounter#3535]
If Personal Server directory is not user directory, the installer shows pop up with the message:

>You are installing Personal Server not in user directory. Make sure you have read/write access rights to the directory: 

This happens in the middle of installation, when I don't expect any more questions and doing something else hoping that Starcounter will be installed when I back. But when I back the installer actually has not done match. It will be much better user experience if this pop up happens at the same time when the other questions. The perfect place for it is when I set the directory for the server.

The problem exists in both 2.x and 2.3

**Comment by k-rus** [comment]
cc @alemoi @andwah

**Comment by alemoi** [comment]
I agree with your suggestion. @andwah can you please add the check from the Level1\src\Starcounter.Installer\Starcounter.InstallerEngine\Components\PersonalServer.cs:295 to your GUI installer? You can remove the whole if there from PersonalServer.cs:295 once you add it to GUI installer. Let me know if any questions.

**Comment by dan31** [comment]
@andwah bump

**Comment by andwah** [comment]
Fixed and pushed to develop.
